bigtable/tests/apply_test.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <bigtable/client/data.h>
+
+int main() try {
+  bigtable::Client client("emulated", "emulated");
+  std::unique_ptr<bigtable::Table> table = client.Open("test-table");
```

Comment:
Nope, there is no Admin API, nor is the Admin API part of the MVP.  I have created #79 for this.  I am inclined to do (2) because it will make this program reusable for testing against production too.  I used command-line args for the first cut.